tap_gorgias/client.py

```diff
@@ -83,9 +83,10 @@ def validate_response(self, response: requests.Response) -> None:
             time.sleep(retry_after)
             raise RetriableAPIError(msg)
         elif 400 <= response.status_code < 500:
+            error_text = json.loads(response.text)
```

Comment:
You can grab the response as a `dict` with `response.json()` so you don't have to parse the `text` yourself.

Another note here that because this catches all http codes between the range 400 and 500 they all might not have the `error` field which might cause the below part of `error_text['error']` to fail with a `KeyError`. This can be worked around by using the `get()` method, which can handle default values in case of missing keys.